content/en/news/releases/1.6.x/announcing-1.6.9/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Istio 1.6.9
+linktitle: 1.6.9
+subtitle: Patch Release
+description: Istio 1.6.9 patch release.
+publishdate: 2020-09-09
+release: 1.6.9
+aliases:
+    - /news/announcing-1.6.9
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.8 and Istio 1.6.9.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** istioctl analyzer to detect when Destination Rules do not specify `caCertificates` ([Istio 25652](https://github.com/istio/istio/issues/25652))
+- **Added** missing `telemetry.loadshedding.*` options to mixer container arguments
+- **Fixed** HTTP match request without headers conflict
+- **Fixed** Istio operator to watch multiple namespaces ([Istio 26317](https://github.com/istio/istio/issues/26317))
```

Comment:
Nit: should these be issue #... Rather than Istio ...?

Reply:
updated